### PR TITLE
change color or menu

### PR DESCRIPTION
**Pull request by BedosSimplice**

### CODE REVIEW

File: src/IKNSA/CMS/BackBundle/Resources/views/page/index.html.twig [hunk 1]
```diff
@@ -1,55 +1,45 @@
 {% extends "IKNSACMSBackBundle::layout.html.twig" %}
 
-{% block body %}
+{% block body %}
```
Comment (khalid-s) [inline src/IKNSA/CMS/BackBundle/Resources/views/page/index.html.twig]: Whitespace

File: src/IKNSA/CMS/BackBundle/Resources/views/page/index.html.twig [hunk 2]
```diff
@@ -1,55 +1,45 @@
...
+                            <a class="btn btn-primary" href="{{ path('iknsa_cms_back_page_show', { 'id': page.id }) }}">show</a><a class="btn btn-primary" href="{{ path('iknsa_cms_back_page_edit', { 'id': page.id }) }}">edit</a>
+                         </td>
+                        </tr>
+                        {% endfor %}
+                        </tbody>
+                    </table>
+                </div>
+            </div>
+        </div>
+{% endblock %}
```
Comment (khalid-s) [inline src/IKNSA/CMS/BackBundle/Resources/views/page/index.html.twig]: EOF

**Comment by BedosSimplice** [comment]
@khalid-s

**Comment by khalid-s** [comment]
:+1:

**Pull request merged**